voice/bxml/callbacks/transferAnswer.md

```diff
@@ -1,19 +1,38 @@
 {% method %}
 ##  Transfer Answer Event – <Transfer> verb
 When processing a [`<Transfer>`](../verbs/transfer.md) verb, this event is sent when a called party (B-leg) answers.  The event is sent to
-  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered.  [`<PlayAudio>`](../verbs/playAudio.md) and/or [`<SpeakSentence>`](../verbs/speakSentence.md) verbs returned by this callback will be
+  the endpoint specified in the `transferAnswerUrl` attribute of the `<PhoneNumber>` tag that answered. Accepted verbs returned by this callback will be
   executed for the called party only. Afterward, the called party will be bridged to the original
   call.
-  
-  It is important to note that no other BXML verbs may be specified after a Transfer Answer Event is called.
+
+| Verb            | Accepted |
+|-----------------|----------|
+| [Conference](../verbs/conference.md)      | No       |
+| [Forward](../verbs/forward.md)         | No       |
+| [Gather](../verbs/gather.md)          | Yes      |
+| [Hangup](../verbs/hangup.md)          | Yes      |
+| [Pause](../verbs/pause.md)           | Yes      |
+| [PauseRecording](../verbs/pauseRecording.md)  | Yes      |
+| [PlayAudio](../verbs/playAudio.md)       | Yes      |
+| [Record](../verbs/record.md)          | Yes      |
+| [Redirect](../verbs/redirect.md)        | Yes      |
+| [ResumeRecording](../verbs/resumeRecording.md) | Yes      |
+| [SendDtmf](../verbs/sendDtmf.md)        | Yes      |
+| [SpeakSentence](../verbs/speakSentence.md)   | Yes      |
+| [StartRecording](../verbs/startRecording.md)  | Yes      |
+| [StopRecording](../verbs/stopRecording.md)   | Yes      |
+| [Transfer](../verbs/transfer.md)        | No       |
```

Comment:
Spacing is weird in this table. Recommend either removing all the whitespace or lining it up. As it is there's some padding but it doesn't line up.